### PR TITLE
SEO Improvements to the Artist page

### PR DESCRIPTION
**Pull request by ashkan18**
# Changes
- Show Artist's full bio and related categories by removing "Read More": https://artsyproduct.atlassian.net/browse/PURCHASE-1569

- Update consignment links to mention consign instead of learn more: 
https://artsyproduct.atlassian.net/browse/PURCHASE-1579

image

cc: @samchieng

### CODE REVIEW

File: src/Components/v2/ArtistBio.tsx [hunk 1]
```diff
@@ -16,20 +15,15 @@ export const MAX_CHARS = {
...
   render() {
+    const { bio } = this.props
     return (
       <Serif size="3">
-        <Media at="xs">{this.renderReadMore("xs")}</Media>
-        <Media greaterThan="xs">{this.renderReadMore("rest")}</Media>
+        <span
+          dangerouslySetInnerHTML={{
+            __html: bio.biography_blurb.text,
+          }}
```
Comment (zephraph) [inline src/Components/v2/ArtistBio.tsx]: Hmmm... do we have to do this? Does the bio have html contents?

Comment (zephraph) [reply src/Components/v2/ArtistBio.tsx]: I think my concern is this opens us up to persisted XSS attacks if a partner can add bio text.

File: src/Apps/Artist/Routes/Overview/index.tsx [hunk 2]
```diff
@@ -94,13 +78,11 @@ export class OverviewRoute extends React.Component<OverviewRouteProps, State> {
...
-                  <Spacer mb={1} />
-                  <Genes artist={artist} />
-                  <Spacer mb={1} />
-                </>
-              )}
+              <>
+                <Spacer mb={1} />
+                <Genes artist={artist} />
+                <Spacer mb={1} />
+              </>
```
Comment (zephraph) [inline src/Apps/Artist/Routes/Overview/index.tsx]: 👍

File: src/Components/v2/ArtistBio.tsx [hunk 3]
```diff
@@ -16,20 +15,15 @@ export const MAX_CHARS = {
...
-    )
-  }
   render() {
+    const { bio } = this.props
     return (
       <Serif size="3">
-        <Media at="xs">{this.renderReadMore("xs")}</Media>
-        <Media greaterThan="xs">{this.renderReadMore("rest")}</Media>
+        <span
+          dangerouslySetInnerHTML={{
```
Comment (ashkan18) [inline src/Components/v2/ArtistBio.tsx]: @zephraph copied this from Palette's `ReadMore`, have no clue what it does, let me know if you have any concerns with this.

Comment (zephraph) [reply src/Components/v2/ArtistBio.tsx]: I talked to @sweir27 about this. We should probably follow up later to make sure we're covering the XSS cases.

**Comment by samchieng** [comment]
Looks good – one spacing thing, I think the extra padding above the bio was always there, but wanted to make it slightly smaller to have 20px between the divider and the bio copy

Zeplin spec here: https://zpl.io/bAnYWj6

**Comment by ashkan18** [comment]
good call @samchieng , fixed it 
image

File: src/Components/v2/ArtistBio.tsx (hunk 3 above)
Comment (mzikherman) [reply src/Components/v2/ArtistBio.tsx]: I think this is about accepting markdown as input (from Artsy admins or partners), and then generating and rendering the HTML (with links, etc.).

Sanitizing the generated HTML is probably well worth a follow up, likely in https://github.com/artsy/metaphysics/blob/ac9c2bbadf84b00ad3bcb5bf271039f7909b334f/src/schema/v1/fields/markdown.ts

**Comment by ashkan18** [comment]
thanks all going to merge 💜

**Comment by artsyit** [comment]
:rocket: PR was released in v21.10.0 :rocket:

**Pull request merged**